src/components/organisms/events/table.tsx

```diff
@@ -0,0 +1,179 @@
+		() =>
+			cn(
+				"resize-handle-horizontal bg-gray-white z-0 h-[97%] cursor-ew-resize self-center rounded-none p-1.5 transition hover:bg-gray-750",
+				{
+					"h-full bg-gray-1100 p-1": !eventId,
+				}
+			),
+		[eventId]
+	);
+
+	const frameClass = useMemo(
+		() =>
+			cn("size-full bg-gray-1100 pb-3 pl-7 transition-all", {
+				"rounded-r-none": !eventId,
+			}),
+		[eventId]
+	);
+
+	const handleSort = useCallback(
+		(key: keyof BaseEvent) => {
+			return (event: React.MouseEvent | React.KeyboardEvent) => {
+				if (event.type === "click" || (event as React.KeyboardEvent).key === "Enter") {
+					requestSort(key);
+				}
+			};
+		},
+		[requestSort]
+	);
+
+	const renderSortableHeader = useCallback(
+		(columnKey: keyof BaseEvent, columnLabel: string) => {
+			return (
+				<div
+					className="group cursor-pointer font-normal outline-none focus:ring-2 focus:ring-blue-500"
+					onClick={handleSort(columnKey)}
+					onKeyDown={handleSort(columnKey)}
+					role="button"
+					tabIndex={0}
+				>
+					{columnLabel}
+					<SortButton
+						className="opacity-0 group-hover:opacity-100 group-focus:opacity-100"
+						isActive={columnKey === sortConfig.key}
+						sortDirection={sortConfig.direction}
+					/>
+				</div>
+			);
+		},
+		[handleSort, sortConfig]
+	);
+
+	const onScroll = useCallback(
+		(event: React.UIEvent<HTMLDivElement>, height: number) => {
+			const { scrollTop } = event.currentTarget;
+			const start = Math.floor(scrollTop / rowHeight);
+			const end = Math.min(sortedEvents.length, Math.ceil((scrollTop + height) / rowHeight) + 1);
+			setVisibleRows({ start, end });
+		},
+		[sortedEvents.length]
+	);
+
+	const renderRows = useCallback(
+		({ height }: { height: number }) => {
+			const start = visibleRows.start;
+			const end = Math.min(visibleRows.end, Math.ceil(height / rowHeight) + start);
+
+			return sortedEvents.slice(start, end).map((event) => (
+				<Tr
+					className="cursor-pointer hover:bg-gray-750"
+					key={event.eventId}
+					onClick={() => navigate(`events/${event.eventId}`)}
+				>
+					<Td>{moment(event.createdAt).format(dateTimeFormat)}</Td>
+					<Td>{event.eventId}</Td>
+				</Tr>
+			));
+		},
+		[sortedEvents, visibleRows, navigate]
+	);
+
+	return (
+		<div className="my-2 flex size-full">
+			<div style={{ width: `${leftSideWidth}%` }}>
+				<Frame className={frameClass}>
+					{loadingEvents ? <Loader isCenter size="xl" /> : null}
+
+					{!loadingEvents && !sortedEvents?.length ? (
+						<div className="mt-10 text-center text-xl font-semibold">{t("history.noEvents")}</div>
+					) : null}
+
+					{!loadingEvents && !!sortedEvents?.length ? (
+						<div className="mt-4 h-full">
+							<AutoSizer>
+								{({ height, width }) => (
+									<div
+										className="overflow-auto"
+										onScroll={(event) => onScroll(event, height)}
+										style={{ width, height }}
+									>
+										<Table className="relative w-full overflow-visible">
+											<THead>
```

Comment:
in the table, I see only the header 
![image](https://github.com/user-attachments/assets/5f9e21d9-d9a8-4a7e-aba4-b02efcdace1d)

Reply:
Try to run a few session with webhook for example.
(Manual run doesn't create event)

Reply:
This is how it looks on my side:
![image](https://github.com/user-attachments/assets/3260c585-b2ce-44fe-ba9e-8fe2e211a611)